mgmt_cli_test.py

```diff
@@ -173,12 +187,47 @@ def test_backup_multiple_ks_tables(self):
         manager_tool = mgmt.get_scylla_manager_tool(manager_node=self.monitors.nodes[0])
         mgr_cluster = manager_tool.add_cluster(name=self.CLUSTER_NAME + '_multiple-ks', db_cluster=self.db_cluster,
                                                auth_token=self.monitors.mgmt_auth_token)
-        self.create_ks_and_tables(10, 100)
+        tables = self.create_ks_and_tables(10, 100)
+        self.log.debug(f'tables list = {tables}')
+        # TODO: insert data to those tables
         backup_task = mgr_cluster.create_backup_task({'location': location_list})
         backup_task.wait_for_status(list_status=[TaskStatus.DONE])
         self.verify_backup_success(bucket=location_list[0].split(':')[1], cluster_id=backup_task.cluster_id)
         self.log.info('finishing test_backup_multiple_ks_tables')
 
+    def test_backup_location_with_path(self):
+        # TODO: add this test to the test_backup
+        self.log.info('starting test_backup_location_with_path')
+        self.update_cred_file()
+        location_list = ['s3:manager-backup-tests-us/path_testing/']
+        manager_tool = mgmt.get_scylla_manager_tool(manager_node=self.monitors.nodes[0])
+        mgr_cluster = manager_tool.add_cluster(name=self.CLUSTER_NAME + '_bucket_with_path',
+                                               db_cluster=self.db_cluster,
+                                               auth_token=self.monitors.mgmt_auth_token)
+        self._generate_load()
+        try:
+            mgr_cluster.create_backup_task({'location': location_list})
+        except ScyllaManagerError as error:
+            self.log.info(f'Expected to fail - error: {error}')
+        self.log.info('finishing test_backup_location_with_path')
+
+    def test_backup_rate_limit(self):
+        # TODO: add this test to the test_backup
+        self.log.info('starting test_backup_rate_limit')
+        self.update_cred_file()
+        location_list = ['s3:manager-backup-tests-us']
+        manager_tool = mgmt.get_scylla_manager_tool(manager_node=self.monitors.nodes[0])
+        mgr_cluster = manager_tool.add_cluster(name=self.CLUSTER_NAME + '_rate_limit', db_cluster=self.db_cluster,
+                                               auth_token=self.monitors.mgmt_auth_token)
+        self._generate_load()
+        rate_limit = ','.join([f'{dc}:{randint(1, 10)}' for dc in self.get_all_dcs_names()])
+        self.log.info(f'rate limit will be {rate_limit}')
+        backup_task = mgr_cluster.create_backup_task({'location': location_list, 'rate-limit': rate_limit})
+        task_status = backup_task.wait_and_get_final_status()
+        self.log.info(f'backup task finished with status {task_status}')
+        self.verify_backup_success(bucket=location_list[0].split(':')[1], cluster_id=backup_task.cluster_id)
```

Comment:
why calling a function under 'self'?
it looks like a better place for this function is as a method of either 'BackupTask' or of 'ManagerCluster'. so there will also  be no need sending the  'cluster_id=backup_task.cluster_id' parameter